src/c++/library/http_client.cc

```diff
@@ -1466,10 +1466,15 @@ InferenceServerHttpClient::Infer(
   // RECV_END will be set.
   auto curl_status = curl_easy_perform(easy_handle_);
   if (curl_status == CURLE_OPERATION_TIMEDOUT) {
-    sync_request->http_code_ = 499;
+    std::cerr << "Curl failed with return code: " << curl_status << std::endl;
```

Comment:
I think we shouldn't log the error message. We should just return the error and it would be up to the user whether they want to log it or not.

Reply:
I originally wrapped it in `if (verbose_)` to log such things in the event user forgets to check, do you think we shouldn't even do that?

Reply:
For errors I think it is not required. The user should always check the error object when using client APIs.

Reply:
removed